repoindex.md

```diff
@@ -37,7 +37,6 @@ organization in GitHub under
 					<li>libpmemlog
 					<li>librpmem
 					<li>libpmempool
-					<li>libvmem (move to its own <strong>vmem</strong> repo is in progress)
```

Comment:
Strictly speaking, yes, it is done, but I wanted to remove this at the moment of the first PMDK release without libvmem.

But I don't have a strong opinion about it.

Reply:
I believe I can agree with you, but.. as the con-argument - they are installed in the system as libvmem, not as part of some "pmdk" package...

Also, "https://pmem.io/pmdk/" still describes libvmem, so we'll have to fix there too.